scripts/index_d_ts

```diff
@@ -2427,16 +2427,20 @@ declare module '@sendbird/uikit-react/ui/MessageStatus' {
 
 declare module '@sendbird/uikit-react/ui/Modal' {
   interface ModalProps {
-    className?: string,
-    onCancel: () => void,
-    onSubmit: () => void,
-    hideFooter?: boolean,
-    disabled?: boolean,
-    type?: string,
+    className?: string;
+    hideFooter?: boolean;
+    disabled?: boolean;
+    type?: string;
     isFullScreenOnMobile?: boolean;
     isCloseOnClickOutside?: boolean;
-    children: React.ReactElement | string,
-    renderHeader?: () => React.ReactElement,
+    children: React.ReactElement | string;
+    /**
+     * Do not use this! onCancel will be deprecated. Use onClose instead.
```

Comment:
Should we include in which version it will be deprecated?

Reply:
Let's say v4, is it okay? @liamcho